app/controllers/questions_controller.rb

```diff
@@ -28,6 +28,7 @@ def short
     @question.build_correct_answer
   end
 
+
```

Comment:
I noticed you added a bunch of extra line-breaks in different places—were they all intended?